guides/mix.lock

```diff
@@ -1,3 +1,7 @@
-%{"earmark": {:hex, :earmark, "1.2.3", "206eb2e2ac1a794aa5256f3982de7a76bf4579ff91cb28d0e17ea2c9491e46a4", [:mix], [], "hexpm"},
-  "ex_doc": {:hex, :ex_doc, "0.16.3", "cd2a4cfe5d26e37502d3ec776702c72efa1adfa24ed9ce723bb565f4c30bd31a", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, repo: "hexpm", optional: false]}], "hexpm"},
-  "fs": {:hex, :fs, "0.9.2", "ed17036c26c3f70ac49781ed9220a50c36775c6ca2cf8182d123b6566e49ec59", [:rebar], []}}
+%{"earmark": {:hex, :earmark, "1.2.3", "206eb2e2ac1a794aa5256f3982de7a76bf4579ff91cb28d0e17ea2c9491e46a4", [:mix], []},
```

Comment:
Please update your Hex client so you don't revert changes in the mix.lock.

Reply:
I'm a little lost here... I've update my hex client. What should I do to "regenerate" the `mix.lock` file so that the parts that didn't change are converted to the format you want? Or should I do it by hand?

Reply:
Reset the file and run `mix deps.get` again:

```sh
git checkout mix.lock
mix deps.get
```

Reply:
Thanks :)

Reply:
Please do the same for the mix.lock in the root.

Reply:
I'm sorry, I missed that one in the diff. Done